### PR TITLE
[GUVNOR-3359] Kie server concurrent timer test

### PR DESCRIPTION
**Pull request by sutaakar**
Disabled until timer concurrency is fixed.

### CODE REVIEW

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/jbpm/TimerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    private static DeploymentScenarioBuilderFactory deploymentScenarioFactory = DeploymentScenarioBuilderFactoryLoader.getInstance();
+    private WorkbenchWithKieServerScenario workbenchWithKieServerScenario;
+    private WorkbenchClient workbenchClient;
+    private KieServerClientProvider kieServerClientProvider;
+    private KieServerMgmtControllerClient kieControllerClient;
+    private GitProvider gitProvider;
+
+    @Before
+    public void setUp() {
```
Comment (jhrcek) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/jbpm/TimerIntegrationTest.java]: This setup / teardown logic looks very similar to other tests in this test suite. How about consolidating it in some test util class?

Comment (jiripetrlik) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/jbpm/TimerIntegrationTest.java]: +1

Comment (sutaakar) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/jbpm/TimerIntegrationTest.java]: @jhrcek good idea, I will prepare something

**Comment by jiripetrlik** [comment]
Still looks good for me.

**Pull request merged**